packages/protocol/types/web3-celo.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'web3-eth' {
```

Comment:
I know we don't want to make protocol dependent on contractkit
Is there any way we can merge these definitions to prevent duplication?